druid/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 
 #![deny(intra_doc_link_resolution_failure, unsafe_code)]
 #![allow(clippy::new_ret_no_self)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
what does this do?

Reply:
I'm fairly certain this is saying: if compiling for docs.rs, do this fancy doc_cfg thing.